src/EditorFeatures/Test2/Rename/RenameNonRenameableSymbols.vb

```diff
@@ -482,5 +482,168 @@ End Class
                 AssertTokenNotRenamable(workspace)
             End Using
         End Sub
+
+#Region "Rename In Tuples"
+
+        <WorkItem(10898, "https://github.com/dotnet/roslyn/issues/10898")>
+        <WorkItem(10567, "https://github.com/dotnet/roslyn/issues/10567")>
+        <WpfFact>
+        <Trait(Traits.Feature, Traits.Features.Rename)>
+        <Trait(Traits.Feature, Test.Utilities.Traits.Features.Tuples)>
+        Public Sub RenameTupleFiledInDeclaration()
+
+            Using workspace = CreateWorkspaceWithWaiter(
+                    <Workspace>
+                        <Project Language="C#" CommonReferences="true" PreprocessorSymbols="__DEMO__">
```

Comment:
We can't specify feature flags properly? @CyrusNajmabadi/@jaredpar weren't one of you looking at this?